gulpfile.js/build.js

```diff
@@ -227,7 +243,14 @@ function buildPrepare(done) {
       lint.lintNode,
       // Build playground and boilerplate that early in the flow as they are
       // fairly quick to build and would be annoying to eventually fail downstream
-      gulp.parallel(buildPlayground, buildBoilerplate, buildSamples, importAll, zipTemplates),
+      gulp.parallel(
+          buildComponentVersions,
```

Comment:
buildComponentVersions also has to run when the develop server is started

Reply:
`gulp bootstrap` should be run after checkout which will add the file